tests/integration/test_ols.py

```diff
@@ -249,3 +249,56 @@ def test_post_question_on_noyaml_response_type() -> None:
                         "The following response was generated without access to reference content:"
                         in response.json()["response"]
                     )
+
+
+def test_post_query_with_question_filter_response_type() -> None:
+    """Check the REST API /v1/query with POST HTTP method with question filter."""
+    config.init_empty_config()
+    config.ols_config.reference_content = ReferenceContent(None)
+    config.ols_config.reference_content.product_docs_index_path = "./invalid_dir"
+    config.ols_config.reference_content.product_docs_index_id = "product"
```

Comment:
setting up an invalid path for reference content seems strange.  Why not just not provide a reference content config at all?  OLS is supposed to still respond to queries even if rag is unavailable